<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DimensionsDialog</class>
 <widget class="QDialog" name="DimensionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimensions de l'espace</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 126, 171);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelLargeur">
       <property name="text">
        <string>Largeur (m) :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputLargeur">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHauteur">
       <property name="text">
        <string>Hauteur (m) :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputHauteur">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelProfondeur">
       <property name="text">
        <string>Profondeur (m) :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputProfondeur">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DimensionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DimensionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
